libethereum/EthereumCapability.cpp

```diff
@@ -206,13 +212,14 @@ class EthereumHostData: public EthereumHostDataFace
                         top = n + step * (numHeadersToSend - 1);
                     }
                     assert(top <= lastBlock && "invalid top block calculated");
-                    blockHash = m_chain.numberHash(static_cast<unsigned>(top)); // override start block hash with the hash of the top block we have
+                    blockHash = m_chain.numberHash(static_cast<unsigned>(
+                        top));  // override start block hash with the hash of the top block we have
```

Comment:
I don't know why you reformatted this and some lines below, but then it's better to move the comment to a separate line above, to make it look nicer
(applies to some cases below in this file, too)

Reply:
@gumb0 😀 https://github.com/ethereum/aleth/pull/5713#issuecomment-518047367